app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
> I don't really understand the lot problem to be honest

I did a bit of investigation and I see a way I could work around the lot treatment differences. I would however need to screw around with the service search code a little bit to make a shared implementation work (it would need modernization - it still uses hard-coded lot names), but the feeling is to just get this in now and look at unification in a future story.